k8s/sonarqube/chart/sonarqube/templates/sonarqube-sts.yaml

```diff
@@ -20,6 +20,18 @@ spec:
         command: ['sh', '-c',
           'until pg_isready -h {{ .Release.Name }}-postgresql-service -p 5432;
           do echo waiting for database; sleep 2; done;']
+      # https://docs.sonarqube.org/latest/requirements/requirements/#header-6
+      # According to the documentation, vm.max_map_count should be greater or equal to 262144
+      - name: increase-max-map-count
+        image: gcr.io/google-appengine/debian9
```

Comment:
Please use the jenkins image - it would already be pulled and republished.

Reply:
will use deployer image